src/remembrall/helpers/wrappers.py

```diff
@@ -1,10 +1,12 @@
 import re
-from classes.customErrors import ShortName, PhoneValidationError, DateFormatError
+from classes.customErrors import ShortName, PhoneValidationError, DateFormatError, AddressValidationError
 
 def input_error(func):
     def inner(*args, **kwargs):
         try:
             return func(*args, **kwargs)
+        except AddressValidationError as ave:
+            return ave.args[0]
```

Comment:
Можешь просто повернути так, воно його все одно перетворить на рядок, як у нижче прикладах
```suggestion
            return ave
```

Reply:
Добре, дякую, буде зроблено.